post-processor/vsphere-template/step_choose_datacenter.go

```diff
@@ -29,7 +29,7 @@ func (s *stepChooseDatacenter) Run(ctx context.Context, state multistep.StateBag
 	if err != nil {
 		err = fmt.Errorf("error finding datacenter %s: %s", s.Datacenter, err)
 		state.Put("error", err)
-		ui.Error(err.Error())
+		ui.Errorf("%s", err)
```

Comment:
nit: Generally speaking we reserve on ui.Errorf for cases where we want to format the outputted string. This change is not an issue but it is not necessary given that you are just displaying the err as is.